demo/manifests/components/cert-manager.jsonnet

```diff
@@ -3,9 +3,28 @@ local kube = import '../vendor/kube-prod-runtime/lib/kube.libsonnet';
 
 local CERT_MANAGER_IMAGE = '';
 
+local add_acme_spec(issuer, obj) =
+  if std.objectHas(issuer.spec, 'acme') then
+    obj {
+      spec+: {
+        acme: {
+          config: [{
+            dns01: {
+              provider: issuer.spec.acme.dns01.providers[0].name,
+            },
+            domains: obj.spec.dnsNames,
+          }],
+        },
+      },
+    }
+  else
+    obj;
+
 upstream_cert_manager {
+  ca_secret_name:: 'ca-key-pair',
+
   // create simple to use certificate resource
-  Certificate(namespace, name, issuer, domains):: kube._Object($.certCRD.spec.group + '/' + $.certCRD.spec.version, $.certCRD.spec.names.kind, name) + {
+  Certificate(namespace, name, issuer, domains):: add_acme_spec(issuer, kube._Object($.certCRD.spec.group + '/' + $.certCRD.spec.version, $.certCRD.spec.names.kind, name) + {
```

Comment:
Maybe worth upgrading to the later cert-manager that removed that acme field?

I guess Let's Encrypt will deprecate <= 0.8.0 in a couple of months